### DISCUSSION TITLE
OpenOOH Venue Taxonomy 1.2 Planning Agenda (2023)

### DISCUSSION THREAD

**Discussion by jayshao** [openooh/venue-taxonomy#130] [Meetings]
Plans for 1.2
* Focus on incremental additions and changes
* Target Q1

Rough timeline:
* 2/15 Public Draft
* 3/15 Final Release
* 3/31 Promotion

**Comment by jayshao** [comment]
A quick pass at what looks like (based on upvotes) the activity has been in the issue tracker:

Requests for Additions (seemingly non-controversial)

* Add: Leisure -> Movie Theaters -> Auditorium #68
* Bug: Incorrect description for "DMVs" category #78
* Add: "Retail" > "Automotive" #88
* Add: "Health and Beauty" > "Indoor Sports Hall" #124
* Add: Office Buildings_Coworking Space #125
* Add: "Retail" > "Home Improvement" #87
* Add: Taxi > Inside/Outside #20
* Add: "Point of Care" > "Hospital" and "Clinic" #104
* Add: Retail > Fueling Stations > Charging Stations #105
* Add: Conference & Convention Centers & Casinos #108
* Add "Restroom" as a category under enclosed spaces where available #99
* Add: Transit ->Travel Plaza #113 (same as * Add: "Billboards" -> "Motorway Service Area" #109?)
* Add: Bus Station #107
* Add: Point of Care > Hospitals #115
* Add: "lobby" for enclosed spaces #98
* Add "Retail" > "Discount Store" #119
* Add "Retail" > "Big Box" #118

Requests for Additions (with seemingly little support)

* Add: Retail > Malls > Normalize Mall Grandchildren #110

Requests for Additions (controversial)

* Pharmacies in Retail Grocery #127 - do we duplicate the taxonomy?
* Add: Subway > Train and train_outside, and Rail > "train" and "train_outside" (like buses) #97 - do trains have outside media?
* Standardization for Residential Apartment Buildings #73 - pending deprecating/removing strings for compatibility
* Add "shopping street", "shopping promenade", "public park" as grandchildren to outdoor.urban_panels #50
* Replace Taxi & Rideshare TV & Top categories with "Cars" or "Automobiles" #100
* Introduce a 0 in front of parent types #126

Tasks

* Should have 101 Guide Document for New DOOH DSPs and SSPs OpenOOH onboarding steps. #75 (Dara)
* Remove deprecated string formats #116
* Align wire protocol with OpenRTB 2.6-202211 #117
* Buyer-Friendly Taxonomy Listing #106 (Nic)

**Reply by davidweinfeld** [reply]
What’s the status of the development of the 1.2 Standards? Is there an expected delivery or activation date?

**Comment by joshbernstein-wu** [comment]
Regarding the delineation between Urban Panels and Window Panels, I do have concerns that by separating Urban Panels from Window Panels (which have virtually the same description) at the same tier will penalize networks like Western Union. We have specifically installed high nit 55" window screens for the purpose of reaching people in urban environments that are either walking or driving by. Keeping them at the same tier will favor the urban panel and would require buyers to have to change their buying behavior/algorithms to add Window Panels which will hurt our revenue. Going to the cookie analogy... this is an opt in not an opt out which is a much harder barrier to cross. IF it is really necessary to create a distinction in the taxonomy, I first believe there should be some hardware distinction but most importantly, I believe the delineations should be childs' to Urban Panels, not siblings. Creating the delineation at the same level is too big a step to make all at once. If it was at the child level, studies over time could analyze how many buyers are selecting to distinguish between the two and if there ends up being a material difference, and buyers have time to adapt to the new taxonomy, future iterations could then fairly be considered for moving it up to a sibling level.

Josh
Director, Media Network
Western Union

**Reply by JoshCooperFullStackPersonal** [reply]
I share the concern around the proposed reclassification of Window Panels. As Josh points out, the practical difference between an Urban Panel and a Window Panel is negligible from a reach and audience experience standpoint. The only distinction is the piece of glass. Treating them as siblings rather than creating a child taxonomy under Urban Panels introduces unnecessary complexity for buyers and risks reducing spend at a moment when consistency and simplicity should be prioritized across the industry. 

Josh Cooper
CEO, Skoop Signage (DOOH CMS)

**Comment by davidweinfeld** [comment]
Attached please find an Open Letter for the Working Group to consider regarding the proposed changes to the Outdoor and Urban Panel taxonomies. 

We are in the process of commissioning a third-party research study to survey the media and advertising community regarding the proposed changes.

[OpenOOH Venue Taxonomy Letter - 10_9_25.pdf](https://github.com/user-attachments/files/22799687/OpenOOH.Venue.Taxonomy.Letter.-.10_9_25.pdf)

**Comment by adi-rds** [comment]
Can the committee also share their cost-benefit impact analysis for these changes? Obviously these changes have an economic impact for buyers and sellers running into the hundreds of millions of dollars. Since, the exchanges don't support grandchild category, only child category changes have economic impact.

For Buyers - would the category changes lead to more choice, less choice or the same amount of choice for their current media plans? 

For Sellers - which sellers remain in the existing child categories? What would be the economic impact for them? For sellers getting steered into new child categories with no latent demand, who would bear the cost of building these categories and driving buyer awareness. It is potentially a multi-year process for buyer education and updating media plans - does the committee have a view on how the costs of this disruption are to be managed?

Clearly, sellers remaining in a whittled down child category would be beneficiaries. Beyond that, what is the benefit to buyers and the broader marketplace? For example, if improved clarity and transparency is the answer, are these child categories sufficiently differentiated and consistent with the stated goal of a venue taxonomy?

**Comment by adi-rds** [comment]
Also for residential are we able to add additional child categories:

Apartments i.e. rentals
Condos i.e. owned
Active Adult / Senior Living - 55+
Student Housing
Affordable Housing

This is how real estate companies themselves split out the verticals in terms of management / reporting, and each captures a very different demographic / audience. We are from TheBulletin.io - we are the largest publisher in the residential space. Having all the varied audiences grouped within a single category is a missed opportunity and may mean advertisers are often advertising on screens that aren't a fit for the audience they want to reach.

**Comment by gotolarivi98-code** [comment]
Really interesting discussion — defining digital-first and hybrid venue types is becoming more important as more industries shift online. Physical venue taxonomies don’t always translate well to digital platforms, and trying to force them into the same structure can lead to ambiguity. It definitely makes sense to consider expanding the taxonomy or creating parallel classifications so both physical and digital environments are properly represented.

While looking into similar categorization challenges in other industries, I recently came across [captain cooks casino 25 free spins](https://captaincooks1.com/)
 and noticed how online platforms often struggle with standardized tagging as well. A more flexible taxonomy could really help keep things consistent across sectors.
